freqtrade/optimize/hyperopt_interface.py

```diff
@@ -180,14 +180,14 @@ def trailing_space(self) -> List[Dimension]:
             # other 'trailing' hyperspace parameters.
             Categorical([True], name='trailing_stop'),
 
-            SKDecimal(0.01, 0.35, decimals=3, name='trailing_stop_positive'),
+            SKDecimal(0.001, 0.1, decimals=3, name='trailing_stop_positive'),
 
             # 'trailing_stop_positive_offset' should be greater than 'trailing_stop_positive',
             # so this intermediate parameter is used as the value of the difference between
             # them. The value of the 'trailing_stop_positive_offset' is constructed in the
             # generate_trailing_params() method.
             # This is similar to the hyperspace dimensions used for constructing the ROI tables.
-            SKDecimal(0.001, 0.1, decimals=3, name='trailing_stop_positive_offset_p1'),
+            SKDecimal(0.02, 0.35, decimals=3, name='trailing_stop_positive_offset_p1'),
```

Comment:
I don't think this change is correct based on how the space is interpreted / used.
for reference: 
https://github.com/freqtrade/freqtrade/blob/be6b1f6f8399b05dcfcddcf63b573052b21beb96/freqtrade/optimize/hyperopt_interface.py#L156-L166

`trailing_stop_positive_offset_p1` does NOT correspond to `trailing_stop_positive_offset` directly, but to `trailing_stop_positive + trailing_stop_positive_offset_p1` (see below).

As such, your new space now requires a profit of between 2% and 35%, before starting to trail with 0.1% to 10%.
0.1% is clearly too tight and will exploit the backtesting mechanics, as it'll then always sell at the best possible position within one candle.

The way it was seemed better / more correct to me, as it required a profit of 1.1%, before trailing with a mininmum of 1%.